libs/text-splitters/pyproject.toml

```diff
@@ -37,6 +37,7 @@ ignore_missing_imports = "True"
 [tool.poetry.dependencies]
 python = ">=3.9,<4.0"
 langchain-core = "^0.3.25"
+beautifulsoup4 = "^4.12.3"
```

Comment:
You can remove this and revert the change to the lock file. We treat almost all deps as optional.